### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by pranavdurai10**
**Proposed changes**:
-Un-used variable v in loop

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @erohmensing will take a look at it as soon as possible ✨

**Comment by pranavdurai10** [comment]
@sara-tagger @erohmensing Hey, any heads-up on my pull request?

File: rasa/core/events/__init__.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def md_format_message(
 
 def first_key(d: Dict[Text, Any], default_key: Any) -> Any:
     if len(d) > 1:
-        for k, v in d.items():
+        for k in d.items():
```
Comment (erohmensing) [inline rasa/core/events/__init__.py:86]: In my opinion it's more confusing to only use the first value from a method that returns two values than to have an unused variable. what about this instead? 
```suggestion
        for k in d.keys():
```

**Review by erohmensing — CHANGES_REQUESTED** [review]
Please be patient for a review when submitting a PR! 

See my comment for a suggestion :)

**Review by erohmensing — APPROVED** [review]
thanks for your first contribution!

**Pull request merged**